groups/index.html

```diff
@@ -3,7 +3,13 @@
 title: BioSchemas Groups
 ---
 
-<h1>Bioschemas Groups</h1>
+<h1>Bioschemas Groups 
+<div class="git-edit" style="float: right;">
+<a target="_blank" href="mailto:leylajael@gmail.com,a.j.g.gray@hw.ac.uk,carolegoble@gmail.com?subject=[Bioschemas] request to create a new group" class="btn btn-bioschema btn-sm" role="button">
```

Comment:
To avoid maintenance issues this should go to the enquiries@bioschemas.org email address, or we could even investigate setting up a mail alias for the management group.